test/puppetlabs/trapperkeeper/bootstrap_test.clj

```diff
@@ -180,3 +269,24 @@ puppetlabs.trapperkeeper.examples.bootstrapping.test-services/foo-test-service ;
       ;; just test that this bootstrap config file can be read successfully
       ;; (ie, this does not throw an exception)
       (bootstrap-with-empty-config ["--bootstrap-config" bootstrap-url]))))
+
+(deftest duplicate-service-definitions
```

Comment:
I saw that you removed the test in a later commit.  Would seem good to remove the `duplicate_services.cfg` file from the PR as well since nothing in this PR uses it.